recipes/libestools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "libestools" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/speech_tools/archive/63ff01938f81443f5a294bc5f9ea6ac6ab38f6b0.tar.gz
+  sha256: 8781ab15ce13fcddcb7df8fd23755a5cb6e4ab7d1a9bdbd741128b990b425496
+
+build:
+  skip: True  # [not linux]
```

Comment:
I personally prefer lower case `true/false` but I don't think YAML cares :)

Reply:
I always understood YAML as kind of  Python-bound :)